game/src/Game/GameStart.gd

```diff
@@ -105,8 +105,35 @@ func _setup_compatibility_mode_paths() -> void:
 		_compatibility_path_list.push_back(actual_base_path + "/mod/" + mod_name)
 
 func _load_compatibility_mode() -> void:
-	if GameSingleton.load_defines_compatibility_mode(_compatibility_path_list) != OK:
+	if GameSingleton.set_compatibility_mode_roots(_compatibility_path_list) != OK:
+		push_error("Errors setting game roots!")
+
+	setup_title_theme()
+
+	if GameSingleton.load_defines_compatibility_mode() != OK:
 		push_error("Errors loading game defines!")
+	
+	SoundSingleton.load_sounds()
+	SoundSingleton.load_music()
+	MusicConductor.add_compat_songs()
+
+func setup_title_theme() -> void:
+	SoundSingleton.load_title_theme()
+
+	MusicConductor.setup_compat_song(SoundSingleton.title_theme)
+
+	var song_names = MusicConductor.get_all_song_names()
+
+	var title_index = song_names.find("The Coronation - Title")
+	if title_index == -1:
+		title_index = song_names.find("thecoronation_titletheme")
```

Comment:
Use `SoundSingleton.get_title_theme()` instead of hardcoding names here as well as in the C++ code?

Reply:
Resolved, but needed another method to be added to search songs by path (as opposed to by title).